src/com/google/javascript/jscomp/CompilerOptions.java

```diff
@@ -1189,12 +1190,16 @@ public boolean getAssumeStaticInheritanceRequired() {
    */
   transient ErrorHandler errorHandler;
 
+  public InstrumentOption instrumentForCoverageOption;
+
   /**
    * Instrument code for the purpose of collecting coverage data.
    */
+  @Deprecated
```

Comment:
Update comment to mention that `instrumentForCoverageOption` should be used insted.

Reply:
Hmm I didn't realize that these were public booleans instead of getter/setter methods.

I was hoping these would be methods we could just make call your new enum setter, so we could preserve the current behavior.

Given that, I'd prefer either:
 - making these options still enable coverage in DefaultPassConfig
 - or if you think that is too unwieldly, just deleting them so that people will get a compiler error if using them instead of a silent behavior change